app/livechat/server/api/v1/visitor.ts

```diff
@@ -69,7 +69,7 @@ API.v1.addRoute('livechat/visitor', {
 	},
 });
 
-API.v1.addRoute('livechat/visitor/:token', {
+API.v1.addRoute(['livechat/visitor/:token', 'voip/visitor/:token'], {
```

Comment:
are we really doing this? I can see 2 main problems to take another approach

- voip is such good name, but and I think we can conflict with future new features, so I would not reserve this path only for omnichannel
- I don think this is a good idea multiple endpoints doing the same, reusing the entire code to do the same thing, if both are the same there is no need to create a new endpoint

Reply:
That is a good comment.  If I understand it right, what you mean is that we should not use voip/visitor combination because in future it may create some problem if the voip scope increases beyond omnichannel tomorrow. But I feel that things still will remain consistent if we increase the scope of Voip. There could be different technologies, different call servers but the framework is going to remain voip. Even though the name is generic (VoiceOverIP), it is explicitly used for VoIP telephony systems. Which means it comes with its own protocol umbrella (SIP, RTP/SRTP, STUN, TURN) or (H323, H245). Though  SIP is getting more popular over H323 and there will gateways who translate H323 to SIP. 

Even though we have a WebRTC way of doing media, (Serves like MediaSoup or JANUS) and the media flows on IP protocol, Conventionally this is not called Voip.  Nevertheless I can understand where you are coming from.  Would you suggest a different name?  (I didnt want to use Voice because the protocol stack supports video too)

Reply:
Hey Amol, let's keep using the normal "visitor" endpoint. No need to create a new route for handling the same 👀

Reply:
Fixed.